callautomation-live-transcription/README.md

```diff
@@ -65,10 +54,8 @@ Open the `.env` file to configure the following settings
 
 ### Run app locally
 
-1. Open a new Powershell window, cd into the `callautomation-live-transcription` folder and run `npm run dev:app`
+1. Open a new Powershell window, cd into the `callautomation-az-openai-voice` folder and run `npm run dev`
```

Comment:
it should be callautomation-live-transcription

Reply:
done